src/adapters/postgres/user-adapter.ts

```
@@ -573,40 +1370,54 @@
-    if (result) {
-      for (let mapData of userCreateDto.tenantCohortRoleMapping) {
-        let cohortData = {
-          userId: result?.userId,
-          cohortId: mapData?.cohortId
+    const result = await this.usersRepository.save(user);
+
+    if (result && userCreateDto.tenantCohortRoleMapping) {
+      for (const mapData of userCreateDto.tenantCohortRoleMapping) {
+        if (mapData.cohortIds) {
+          for (const cohortIds of mapData.cohortIds) {
+            let query = `SELECT * FROM public."CohortAcademicYear" WHERE "cohortId"= '${cohortIds}' AND "academicYearId" = '${academicYearId}'`;
```

ESLint

[error] 'query' is never reassigned. Use 'const' instead.

(prefer-const)